cfglib.py

```diff
@@ -179,11 +179,15 @@
     cfg.StrOpt('default_availability_zone', default="nova",
                help="Availability zone to use for VM provisioning, in case "
                     "source cloud zones do not match destination"),
-    cfg.StrOpt('ephemeral_copy_backend', default="rsync",
-               help="Allows to choose how ephemeral storage is copied over "
-                    "from source to destination. Possible values: 'rsync' "
-                    "(default) and 'scp'. scp seem to be faster, while rsync "
-                    "is more reliable"),
+    cfg.StrOpt('copy_backend', default="rsync",
+               help="Allows to choose how ephemeral storage and cinder "
+                    "volumes are copied over from source to destination. "
+                    "Possible values: 'rsync' (default), 'scp' and 'bbcp'. "
+                    "scp seem to be faster, while rsync is more reliable, "
+                    "but bbcp works in multithreading mode."),
+    cfg.BoolOpt('copy_with_md5_verification', default=True,
```

Comment:
Let's use this option to control `rsync` and `bbcp` behavior: if this option is set, use corresponding checksum option